.github/workflows/build.yaml

```diff
@@ -8,88 +8,88 @@ jobs:
       fail-fast: false
       matrix:
         include:
-        - os: ubuntu-22.04
-          platform: linux
-        - os: macos-latest
-          platform: macos
-          preset: release-macos
-          cpack: true
-        - os: windows-latest
-          platform: windows-msvc
-          preset: release-msvc
-          cpack: true
-          useQtAction: true
-        - os: windows-latest
-          platform: windows-mingw
-          preset: release-ninja
+          - os: ubuntu-22.04
+            platform: linux
+          - os: macos-latest
+            platform: macos
+            preset: release-macos
+            cpack: true
+          - os: windows-latest
+            platform: windows-msvc
+            preset: release-msvc
+            cpack: true
+            useQtAction: true
+          - os: windows-latest
+            platform: windows-mingw
+            preset: release-ninja
     steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-    - name: Setup
-      shell: bash
-      run: |
-        setupScript='ci/${{ matrix.platform }}/setup.sh'
-        [ ! -f "$setupScript" ] || "$setupScript"
-      env:
-        MACOS_ASC_API_KEY: ${{ secrets.MACOS_ASC_API_KEY }}
-        MACOS_CODE_SIGN_KEY_BASE64: ${{ secrets.MACOS_CODE_SIGN_KEY_BASE64 }}
-        MACOS_KEYCHAIN_PASSWORD: ${{ secrets.MACOS_KEYCHAIN_PASSWORD }}
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup
+        shell: bash
+        run: |
+          setupScript='ci/${{ matrix.platform }}/setup.sh'
+          [ ! -f "$setupScript" ] || "$setupScript"
+        env:
+          MACOS_ASC_API_KEY: ${{ secrets.MACOS_ASC_API_KEY }}
+          MACOS_CODE_SIGN_KEY_BASE64: ${{ secrets.MACOS_CODE_SIGN_KEY_BASE64 }}
+          MACOS_KEYCHAIN_PASSWORD: ${{ secrets.MACOS_KEYCHAIN_PASSWORD }}
 
-    - name: Install Qt
-      uses: jurplel/install-qt-action@v3
-      if: ${{ matrix.useQtAction }}
-      with:
-        version: '6.6.2'
-        arch: 'win64_msvc2019_64'
-        archives: 'qtbase qttools opengl32sw d3dcompiler_47'
-        extra: '--external 7z'
-        cache: true
-        setup-python: false
-    - name: Prepare MSVC environment
-      uses: ilammy/msvc-dev-cmd@v1
-      if: ${{ matrix.platform == 'windows-msvc' }}
-      with:
-        arch: x64
-        toolset: 14.29
+      - name: Install Qt
+        uses: jurplel/install-qt-action@v3
+        if: ${{ matrix.useQtAction }}
+        with:
+          version: "6.6.2"
+          arch: "win64_msvc2019_64"
+          archives: "qtbase qttools opengl32sw d3dcompiler_47"
+          extra: "--external 7z"
```

Comment:
yes/no, it will likely work without quotes too but in general quotes are wanted - also shown at the official GHA examples etc so won't hurt doing proper gha/yml

Reply:
well, we don't use them in other places. Either they should be used everywhere or nowhere at all.